documentation/development-processes-checklists.md

```diff
@@ -1,11 +1,10 @@
 ## Naming Convention
-We consider there to be essentially three basic categories of changes:
-- `bugfix` (or `hotfix`): for change that fix a documented defect of the simulator
-    - includes GitHub labels `bugfix` (regular bugfixes) and `hotfix` (critical bug fix pushed directly to production)
-- `feature`: for changes that add or notably improve an existing feature
-    - includes GitHub labels `airport work` (new airports only), `enhancement`, and of course `feature`
-- `refactor`: for changes that neither add features nor fix bugs, like code reorganization
-    - includes GitHub labels `airport work` (all airport updates), `devops`, `documentation`, and of course `refactor`
+Each issue and corresponding pull request will be placed in one of the following categories:
+- `feature`: addition of a _completely new feature_
+- `enhancement`: notable _improvement or expansion upon an existing feature_
+- `refactor`: code changes that _neither fix a defect nor add functionality_
+- `bugfix` and `hotfix`: fix a _defect of the simulator itself_
```

Comment:
what makes a `bugfix` a `bugfix` and a `hotfix` a `hotfix`?  each should have their own line with description.

Reply:
you're right. not sure why I was trying to avoid it. Done!